jest.conf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "preset": "./node_modules/carbon-factory/jest.conf.json",
+  "notify": false,
+  "setupFiles": [
+    "./src/__spec_helper__/index.js"
+  ]
```

Comment:
👍  but you will need to also update the `package.json` scripts to tell the test commands where the jest config is

https://github.com/Sage/carbon/blob/master/package.json#L10-L11

will both need ` --config=./jest.conf.json` adding to it